libbeat/common/mapstr_test.go

```diff
@@ -64,7 +63,7 @@ func TestMapStrCopyFieldsTo(t *testing.T) {
 	c := MapStr{}
 
 	err := m.CopyFieldsTo(c, "dd")
-	assert.Equal(nil, err)
+	assert.Error(err)
```

Comment:
Didn't the previous check assert that Error is nil, and now it verifies that it is no error? The equivalent would be `assert.NoError(err)`?